lonestar/analytics/cpu/connected-components/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-add_executable(connected-components-cpu ConnectedComponents.cpp)
+add_executable(connected-components-cpu connected_components_cli.cpp)
 add_dependencies(apps connected-components-cpu)
 target_link_libraries(connected-components-cpu PRIVATE Galois::shmem lonestar)
 install(TARGETS connected-components-cpu DESTINATION "${CMAKE_INSTALL_BINDIR}" COMPONENT apps EXCLUDE_FROM_ALL)
-add_test_scale(small connected-components-cpu NO_VERIFY INPUT rmat15 INPUT_URI "${BASEINPUT}/propertygraphs/rmat15_symmetric" "-symmetricGraph" "-algo=LabelProp")
+add_test_scale(small connected-components-cpu NO_VERIFY INPUT rmat15 INPUT_URI "${BASEINPUT}/propertygraphs/rmat15_symmetric" "-symmetricGraph" "-algo=ConnectedComponentsPlan::kLabelProp")
```

Comment:
I'm not actually sure how this is passing. I think a refactor got out of hand and changed this.

Reply:
I don't understand. Why shouldn't it work?

Reply:
it has `-algo=ConnectedComponentsPlan::kLabelProp` but command line args expect `-algo=LabelProp`.

Reply:
ohh. This is was correct but you are right we should change this to use -algo=LabelProp. This also needs to be fixed in bfs and sssp. Use `clEnumValN` instead of `clEnumVal`